src/utils.rs

```diff
@@ -50,3 +54,42 @@ pub fn parse_gj_messages_response(meow: String) -> Result<Vec<GDMessage>, Generi
         })
         .collect()
 }
+
+pub async fn checkdiscordusername(mut conn: Connection<Db>, discord_snowflake: String, id: u32) {
```

Comment:
style: rename this to check_discord_username

Reply:
in hindsight: this entire function should be ran repeatedly on all users in the database in a tokio task and not in a request handler, to avoid very slow server responses because its blocking waiting for the discord API query

also: every unwrap in this function will cause the entire request to fail, since it'd cause a panic and terminate the request handler - this is why i want proper error handling